File: t/PACSFramework/VB/MedicalWorkstationMainDemo/UI/ImagesDownloadDialog.vb
' *************************************************************
' Copyright (c) 1991-2019 LEAD Technologies, Inc.              
' All Rights Reserved.                                         
' *************************************************************
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports System.Diagnostics
Imports Leadtools.Demos.Workstation.Configuration

Namespace Leadtools.Demos.Workstation
   Partial Public Class ImagesDownloadDialog
	   Inherits Form
	  Public Sub New()
		 InitializeComponent()

		 Me.Text = ConfigurationData.ApplicationName
	  End Sub

	  Private Sub linkLabel1_LinkClicked(ByVal sender As Object, ByVal e As LinkLabelLinkClickedEventArgs) Handles linkLabel1.LinkClicked
		 Process.Start ("ftp://ftp.leadtools.com/pub/3d/")
	  End Sub
   End Class
End Namespace

File: t/PACSFramework/MedicalWebViewer/WebViewerConfiguration/ImagesDownloadDialog.cs
// *************************************************************
// Copyright (c) 1991-2019 LEAD Technologies, Inc.              
// All Rights Reserved.                                         
// *************************************************************
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Diagnostics;
using System.Configuration;

namespace Leadtools.Demos
{
   public partial class ImagesDownloadDialog : Form
   {
      public ImagesDownloadDialog(string appName)
      {
         InitializeComponent();

         this.Text = appName;
      }

      private void linkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
      {
         Process.Start ( @"ftp://ftp.leadtools.com/pub/3d/" ) ;
      }

      private void _btnOK_Click(object sender, EventArgs e)
      {
      }

      public bool DoNotShowAgainCheckBox
      {
         get
         {
            return _chkBoxDontShowThisAgain.Checked;
         }

         set
         {
            _chkBoxDontShowThisAgain.Checked = value;
         }
      }

      public bool VisibleCheckBox
      {
         get
         {
            return _chkBoxDontShowThisAgain.Visible;
         }
         set
         {
            _chkBoxDontShowThisAgain.Visible = value;
         }
      }

   }
}
